src/overlays/actors/ovl_Bg_Jya_Bombiwa/z_bg_jya_bombiwa.c

```diff
@@ -9,7 +10,9 @@ void BgJyaBombiwa_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgJyaBombiwa_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgJyaBombiwa_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void BgJyaBombiwa_SetupDynaPoly(BgJyaBombiwa* this, GlobalContext* globalCtx, void* arg2, DynaPolyMoveFlag flag);
+void BgJyaBombiwa_InitCollider(BgJyaBombiwa* this, GlobalContext* globalCtx);
+
```

Comment:
In general we avoid forward declaring functions unless necessary, which isn't the case here.
```suggestion
```